esrally/metrics.py

```diff
@@ -123,8 +123,7 @@ def __init__(self, cfg):
         self._config = cfg
         host = self._config.opts("reporting", "datastore.host")
         port = self._config.opts("reporting", "datastore.port")
-        # poor man's boolean conversion
-        secure = self._config.opts("reporting", "datastore.secure") == "True"
+        secure = str(self._config.opts("reporting", "datastore.secure")).capitalize() == "True"
```

Comment:
I think we could even use `convert.to_bool()` here (similarly to treatment of other booleans)?

Reply:
Great suggestion! I've switch to that now.